server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/SetDefaultTemplateAction.java

```diff
@@ -90,7 +89,7 @@ private void doHandle(SetDefaultTemplateWsRequest request) {
       PermissionTemplateDto template = findTemplate(dbSession, request);
       checkGlobalAdmin(userSession, template.getOrganizationUuid());
       validateQualifier(qualifier, resourceTypes);
```

Comment:
I didn't know that the DEV qualifiers was dropped. Indeed, we need to check, at least at runtime the qualifiers available. ResourceTypes does many things, so there's place for simplification. I'm fine with a proper description and hardcoded list. You must mention that views are searchable only with the Governance plugin